index.html

```diff
@@ -62,6 +34,40 @@
           wgURI:        "https://wiki.duraspace.org/display/FEDORAAPI/Fedora+Specification",
           wgPublicList: "https://groups.google.com/forum/#!forum/fedora-community",
           otherLinks: [{
+            key: "Contributors",
```

Comment:
@acoburn I thought it would be better to have the contributors listed at the top, particularly given how involved you and some of the others listed here have been.  Let me know if you feel strongly that the contributor list should be in the acknowledgements section.